org.eclipse.scanning.test/src/org/eclipse/scanning/test/epics/PVDataSerializationTest.java

```diff
@@ -105,6 +105,7 @@ public void TestArrayGenerator() throws Exception {
 		Structure expectedCompGenStructure = fieldCreate.createFieldBuilder().
 				addArray("mutators", union).
 				add("duration", ScalarType.pvDouble).
+				add("continuous", ScalarType.pvBoolean).
 				addArray("generators", union).
 				addArray("excluders", union).
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "excluders" 50 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)